gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zip
```

Comment:
Let's use the Gradle version which is currently set for, say, `kanban`.

There are some changes in Gradle 8, which are not yet compatible with Spine 1.x.